README.md

```diff
@@ -57,6 +57,8 @@ This file contains a number of front-end interview questions that can be used wh
 * Consider HTML5 as an open web platform. What are the building blocks of HTML5?
 * Describe the difference between cookies, sessionStorage and localStorage.
 * Describe the difference between `<script>`, `<script async>` & `<script defer>`.
+* Why is it generally a good idea to position CSS `<link>`s between `<head></head>` and JS `<script>`s just before `</body>`? 
+  * Do you know any exceptions for this rule?
```

Comment:
Can you reword to something a bit more general and open (example: `Are there any exceptions?`) since this really isn't an actual `rule`. Also, can you put both sentences on the same line.